src/components/container/MyDataHelpsAssistant/MyDataHelpsAssistant.css

```diff
@@ -0,0 +1,190 @@
+:root {
```

Comment:
Would remove all these as we have CSS vars for various colors already in MyDataHelpsUI: https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/helpers/globalCss.ts < please use those